.github/workflows/gh-pages-cleanup.yml

```diff
@@ -0,0 +1,39 @@
+name: GitHub Pages Cleanup
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  push:
+    branches:
+      - dev/DS-788-cleanup # test
+
+jobs:
+  cleanup:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: gh-pages
+          token: ${{ secrets.GH_REPO_TOKEN }}
```

Comment:
@meriouma 
Étrange, j'obtiens `Error: Input required and not supplied: token`. J'ai suivi [ceci](https://github.com/EndBug/add-and-commit?tab=readme-ov-file#about-tokens) et j'ai pris le même token qu'on utilise pour le déploiement.
Tu peux valider que le _secret_ existe réellement.

Reply:
Le secret existe seulement pour l'env `github-pages`, il te manque `environment: github-pages`. Tu peux regarder le step `storybook_deploy`

Reply:
Peux-je, en tant que simple utilisateur dépourvu des pouvoirs d'administration, explorer les environnements et leurs secrets ?